src/descriptor/template.rs

```diff
@@ -70,12 +70,12 @@ pub trait DescriptorTemplate {
 /// Turns a [`DescriptorTemplate`] into a valid wallet descriptor by calling its
 /// [`build`](DescriptorTemplate::build) method
 impl<T: DescriptorTemplate> ToWalletDescriptor for T {
```

Comment:
And this one too, perhaps `IntoWalletDescriptor`.

Reply:
Yes good call, fixed.